pkgs/applications/audio/bitwig-studio/bitwig-studio3.nix

```diff
@@ -2,11 +2,11 @@
 
 bitwig-studio1.overrideAttrs (oldAttrs: rec {
   name = "bitwig-studio-${version}";
-  version = "3.1.3";
+  version = "3.2.1";
 
   src = fetchurl {
     url = "https://downloads.bitwig.com/stable/${version}/bitwig-studio-${version}.deb";
-    sha256 = "11z5flmp55ywgxyccj3pzhijhaggi42i2pvacg88kcpj0cin57vl";
+    sha256 = "0lcqm4hbkz2d0dmh4ljix0r9cpqpqnjldm3r7f6dg54pcyr9s9c2";
```

Comment:
```suggestion
    sha256 = "10zb78n75nbriyjah0m3syv3rv7qwbmj590z24hss7lifa3rs784";
```

Reply:
3.2.2 is out already, so this PR might be updated accordingly, here is the right hash (which might save some time)